android/quest/build.gradle.kts

```
@@ -269,6 +269,13 @@ android {
       manifestPlaceholders["appLabel"] = "ZEIR"
     }
 
+    create("gizeir") {
```

Let's rename to the camel Cased naming convention [documented here](https://docs.opensrp.io/engineering/android-app/automated-releases#flavors)
```suggestion
    create("gizEir") {
```